src/DynamoCoreWpf/ViewModels/Watch3D/HelixWatch3DViewModel.cs

```diff
@@ -761,6 +760,13 @@ public override void DeleteGeometryForIdentifier(string identifier, bool request
             DeleteGeometries(geometryModels, requestUpdate); 
         }
 
+        protected override void OnRenderPackagesUpdated(NodeModel node, IEnumerable<IRenderPackage> packages)
```

Comment:
@mjkkirschner I don't think we need to override `OnRenderPackagesUpdated` here anymore as we have already made the necessary changes to it in the base class above.